routes/userMigrations.js

```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const router = express.Router();
+const authenticate = require("../middlewares/authenticate");
+const authorizeRoles = require("../middlewares/authorizeRoles");
+const { SUPERUSER } = require("../constants/roles");
+const migrations = require("../controllers/userMigrations");
+
+router.patch("/userDefaultColor", authenticate, authorizeRoles([SUPERUSER]), migrations.addDefaultColors);
```

Comment:
we should typically use lowercase words separated by hyphens in route names instead of came case

Reply:
ok thanks, i will update this